### ISSUE TITLE
[FEATURE] Reskinning

### ISSUE THREAD

**Issue by jonomnom** [OmakaseaNFT/love-game#15]
I have a feature request that I am currently working on with Jhinnbay, to add a new skin to the love game.  This would also be laying out the groundwork for future skins so that it would be easier to create new skins and add a new skin.

Users will be able to visit love.game and if they get bored of the current look, they can switch it up to get love.game a fresh look.

This update would include

- Documentation on how to add new skins
- Reskin with customizable icons and fonts and color themes
- A new reskin - see https://twitter.com/i/status/1677626920282779648

( side note: I can't create labels on this PR. The gear icon does not show up.)

**Comment by jqmwa** [comment]
![image](https://github.com/OmakaseaNFT/love-game/assets/95416268/bf011d7b-9e9b-405e-a070-0a7aa2bc768c)

**Comment by jonomnom** [comment]
Gentle bump @EtDu

**Comment by EtDu** [comment]
Yeah it'd be cool to have a way to choose which skin you want for the entire site

**Comment by EtDu** [comment]
Who would be programming this?

**Comment by jonomnom** [comment]
Jhinnbay made the designs and I would coding it up. Already started here: https://github.com/jhinnbay/slavecoin . 
 You can see the basic idea here: https://twitter.com/jhinnbay/status/1678228585755344898

Potentially we can have premium skins also that burn love.

**Comment by EtDu** [comment]
Dope! Can you plz make a PR with that fork?

**Comment by jonomnom** [comment]
Will do this weekend. I need to update some colors and create a better looking file themes tab. Will create a PR when it is ready.

**Comment by jqmwa** [comment]
Yo E, I finished the design for the NFT to give access to the theme and jono has coded up the theme, took us awhile but we got it.

Now we wanted to charge a fee for the theme... but we want to have a fair price, we also want to consider splitting revenue with you and the team.

Do you have any ideas or suggestions for us?

[image](https://github.com/OmakaseaNFT/love-game/assets/95416268/48e19d06-bb47-4ebb-951e-7412016c2817)

**Comment by jqmwa** [comment]
Yo```

```

**Comment by EtDu** [comment]
Heyo, just coming back online here

**Comment by EtDu** [comment]
Ah so you want to sell NFTs to give access to the themes?

**Comment by jqmwa** [comment]
Someone in the comments mentioned it, and we thought about it, but really i'd just rather push the code free tbh, it's been so long and idk how people would feel about paying. It also adds another layer of complexity. 

I say we just ship the theme for free.

**Comment by EtDu** [comment]
Ok so hold on, you have one theme that's been made, but its setup to allow for multiple themes right?

**Comment by EtDu** [comment]
I say we ship the first theme for free, then new themes should be paid

**Comment by EtDu** [comment]
Ok so next steps here, plz open a pr against your fork

**Comment by jonomnom** [comment]
#26 @EtDu Thank you! Here is the PR.

**Comment by jonomnom** [comment]
Also, I will be AFK for the 2 weeks.

**Comment by jonomnom** [comment]
Also, I will be AFK for the next 2 weeks.
